### ISSUE TITLE
Fedora Silverblue support?

### ISSUE THREAD

**Issue by dumprop** [MusiKid/fancy#13]
Fedora Silverblue is an immutable desktop operating system and there no dnf

The main reason is that it wouldn’t work on Silverblue — and part of its functionality was replaced by the rpm-ostree command. Users can overlay the traditional packages by using the rpm-ostree install PACKAGE. But it should only be used when there is no other way. This is because when the new system images are pulled from the repository, the system image must be rebuilt every time it is altered to accommodate the layered packages, or packages that were removed from the base OS or replaced with a different version. 

From our point of view the future of the desktop involves the immutable OS. It’s safest for the user, and Android, ChromeOS, and the last macOS Catalina all use this method under the hood. For the Linux desktop there are still problems with some third party software that expects to write to the OS. HP printer drivers are a good example.

Another issue is how parts of the system are distributed and installed. Fonts are a good example. Currently in Fedora they’re distributed in RPM packages. If you want to use them, you have to overlay them and then restart to the newly created image that contains them.

There is a possibility that the Silverblue will replace the regular Workstation.

Source: https://fedoramagazine.org/what-is-silverblue/

**Comment by saidsay-so** [comment]
I maybe misunderstood it, but the preferred way to install packages on Silverblue is through Flatpak. I don't think that it will work for the service, as it needs to communicate with the ACPI, which for now requires privileged access to the system (like when they talk about printer drivers).

**Comment by dumprop** [comment]
> I maybe misunderstood it, but the preferred way to install packages on Silverblue is through Flatpak. I don't think that it will work for the service, as it needs to communicate with the ACPI, which for now requires privileged access to the system (like when they talk about printer drivers).

Yes, I think same too (currently I still use workstation, but probably silverblue will replace it in future)

**Comment by saidsay-so** [comment]
There is no clear roadmap yet for Silverblue, so I think it will be better to focus on RPM for now (but I will eventually add support if the roadmap becomes more explicit).
